docs-gatsby/src/pages/methodology/eventstorming.mdx

```diff
@@ -95,7 +95,7 @@ The second type of boundary is a subject boundary. You can detect a subject boun
 
 You can delineate these different sets of simultaneous event streams by applying blue painter’s tape horizontally, dividing the board into different swim lanes.
 
-Below is an example of a set of ordered domain events with pivotal events and subject swim lanes indicated.  This example comes from applying event storming to the domain of container shipping process and is discussed in more detail in the [container shipment analysis example](https://github.com/ibm-cloud-architecture/refarch-kc/blob/master/analysis/readme.md). When the reefer container is plugged to the Vessel, it starts to emit telemetries, we change context.
+Below is an example of a set of ordered domain events with pivotal events and subject swim lanes indicated.  This example comes from applying event storming to the domain of container shipping process and is discussed in more detail in the [container shipment analysis example](https://github.com/ibm-cloud-architecture/refarch-kc/blob/master/analysis/readme/). When the reefer container is plugged to the Vessel, it starts to emit telemetries, we change context.
```

Comment:
This should point to https://ibm-cloud-architecture.github.io/refarch-kc/analysis/readme